tests/clients/test_experiment.py

```diff
@@ -92,6 +99,30 @@
     "endedAt": RUN_ENDED_AT_STRING,
     "deletedAt": DELETED_AT_STRING,
 }
+TAG = Tag(key="tag-key", value="tag-value")
+TAG_BODY = {"key": "tag-key", "value": "tag-value"}
+
+OTHER_TAG = Tag(key="other-tag-key", value="other-tag-value")
+OTHER_TAG_BODY = {"key": "other-tag-key", "value": "other-tag-value"}
+
+PARAM = Param(key="param-key", value="tag-value")
```

Comment:
`"param-value"` (and on the line below)?

Reply:
👍

Reply:
👍